xesmf/frontend.py

```diff
@@ -369,7 +370,7 @@ def _compute_weights(self):
         esmf_regrid_finalize(regrid)  # only need weights, not regrid object
         return w
 
-    def __call__(self, indata, keep_attrs=False):
+    def __call__(self, indata, keep_attrs=False, adaptative_masking=False):
```

Comment:
I'm not convinced adaptative/adaptive is the right term for this. 
Other options: 
- dynamic_mask
- flexible_mask
- mask_threshold